yoti-sdk-api/src/main/java/com/yoti/api/client/shareurl/policy/DynamicPolicy.java

```diff
@@ -94,6 +94,15 @@ public Object getIdentityProfile() {
         return identityProfile;
     }
 
+    /**
+     * Defines a required advanced identity profile.
+     *
+     * @return AdvancedIdentityProfile
+     */
```

Comment:
These type of javadocs are useless... it is basically just parroting what the method name already says.

Also the summary doesn't even make sense tbh "**Defines** a required...", it's a `get` how does it _define_ anything?

Drop it. Or re-write it so that it provides _more_ information that isn't already embedded in the method name.

Reply:
Going to drop all of them. We do not provide any for `v2` and I do agree the value there is close to 0